docs/development/bump-k8s-go.md

```
@@ -0,0 +1,535 @@
+| 1.33      | 4.20        |
+| 1.34      | 4.21        |
+| 1.35      | 4.22        |
+| 1.36      | 5.0         |
+
+Call this `OCP_VERSION`. This determines the builder image tags.
+
+### 1e. Builder image availability
+
+Verify that CI builder images exist for the target Go version and OCP release:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)